docs/website/layouts/partials/docs/navbar.html

```diff
@@ -59,17 +60,8 @@
 
       {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Miscellaneous" "section" $misc "pageUrl" $pageUrl "version" $version ) }}
 
+      {{ partial "docs/navbar-section.html" ( dict "ctx" . "sectionTitle" "Support" "section" $support "pageUrl" $pageUrl "version" $version ) }}
       <br />
-
-      <span class="navbar-item is-size-5 has-text-weight-light is-uppercase">
-        Support
-      </span>
-      <a class="navbar-item" href="/support">
-        Enterprise and Commercial
-      </a>
-      <a class="navbar-item" href="/community">
-        Community
-      </a>
     </div>
```

Comment:
So the way our multi-version docs are built is as follows:

- the previous version's `docs/content` is checked out into `docs/website/public/v<VERSION>/`
- everything is rendered using the templates, partials, etc, from the main branch

As a consequence, we'll remove the Support/Enterprise/Community links for all previous versions -- they didn't have the pages added here.

So... if you wouldn't mind, adding some conditional around this change here would be great. Like, check $version, see if it's smaller or equal to 0.41.0. Or, if you're good with hugo, check if the pages you've added (community, enterprise) exist, and keep the old links accordingly.

Reply:
Got it. Will look into it.